documentation/0000-template-installation-node-on-mac/0000-template-installation-node-on-mac.md

```diff
@@ -0,0 +1,35 @@
+- **Start Date:** 2018-08-18
+- **PR:** 
+- **Issue:** [#60](https://github.com/nodejs/website-redesign/issues/60)
+- **Keywords:** install node mac 
+- **Summary:** Instruction how to install node.js on MacOS using Homebrew

```

Comment:
I would _highly_ recommend __not__ doing this. The major problem with this type of installation is that it would require `sudo`  to install packages. Let's recommend nvm or nvs.